client/main.lua

```diff
@@ -293,8 +294,8 @@ CreateThread(function()
             local hours = GetClockHours()
             if hours >= Config.MinimumTime or hours <= Config.MaximumTime then
                 if not inside then
-                    for k, v in pairs(Config.Houses) do
-                        dist = #(PlayerPos - vector3(Config.Houses[k]["coords"]["x"], Config.Houses[k]["coords"]["y"], Config.Houses[k]["coords"]["z"]))
+                    for k, _ in pairs(Config.Houses) do
```

Comment:
You can remove this underscore completely

Reply:
The underscore is there to indicate that it is not used, why would it be removed?